includes/donors/class-give-donor-wall.php

```diff
@@ -356,8 +356,8 @@ private function get_query_param( $atts = [] ) {
 		$query_atts['orderby']       = in_array( $atts['orderby'], $valid_orderby ) ? $atts['orderby'] : 'post_date';
 		$query_atts['limit']         = $atts['donors_per_page'];
 		$query_atts['offset']        = $atts['donors_per_page'] * ( $atts['paged'] - 1 );
-		$query_atts['form_id']       = implode( '\',\'', explode( ',', $atts['form_id'] ) );
-		$query_atts['ids']           = implode( '\',\'', explode( ',', $atts['ids'] ) );
+        $query_atts['form_id']       = $this->split_string(implode('\',\'', explode(',', $atts['form_id'])), 'absint');
```

Comment:
Locally this is producing an error in that it breaks the query. This line should be resulting in something like `1','2','3`, which is used directly in the query.

Using `$this->split_string` here results in an `array`.

Reply:
Instead we can just introduce `array_map('absint', ...)` after we `explode` and before we `implode`.

```
implode( '\',\'', array_map( 'absint', explode( ',', $atts['form_id'] ) ) );
```

Reply:
```suggestion
        $query_atts['form_id'] = implode( '\',\'', array_map( 'absint', explode( ',', $atts['form_id'] ) ) );
```